examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/MultiAdminClientFragment.kt

```diff
@@ -209,13 +211,13 @@ class MultiAdminClientFragment : Fragment() {
     deviceController.readAttributePath(
       object : ReportCallback {
         override fun onReport(nodeState: NodeState?) {
-          val value =
+          val tlv =
```

Comment:
https://github.com/project-chip/connectedhomeip/pull/28639
you can dump the json directly for message display soon

Reply:
The reason for this change is that the simple value is not shown in json format.